main_routers/jukebox_router.py

```
@@ -0,0 +1,1156 @@
+        # 写入配置文件
+        with open(export_dir / "config.json", "w", encoding="utf-8") as f:
+            json.dump(export_data, f, ensure_ascii=False, indent=2)
+        
+        # 打包为 zip
+        zip_path = temp_path / "jukebox_export.zip"
+        with zipfile.ZipFile(zip_path, "w", zipfile.ZIP_DEFLATED) as zf:
+            for file_path in export_dir.rglob("*"):
+                if file_path.is_file():
+                    zf.write(file_path, file_path.relative_to(export_dir))
+        
+        # 读取 zip 文件到内存（避免临时目录被清理后文件丢失）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)